requirements.txt

```diff
@@ -10,4 +10,7 @@ sphinxcontrib-serializinghtml==1.1.4
 sphinx-gallery
 matplotlib
 pillow
-scikit-learn
\ No newline at end of file
+scikit-learn
+pandas
+plotly>=4.0.0
+lightgbm
```

Comment:
The sphinx directive `plotly` seems to be missing. Could you add `sphinx-plotly-directive` to `requirements.txt` and `source/conf.py`?

It was introduced by https://github.com/optuna/optuna/pull/1944, and included in v2.4.0.

c.f.,
https://github.com/optuna/optuna/blob/bf60fab2473599188c2722fff1f743d372412331/docs/source/conf.py#L59
https://github.com/optuna/optuna/blob/bf60fab2473599188c2722fff1f743d372412331/setup.py#L74

The warning message related to the missing directive is as follows:

```
reading sources... [100%] tutorial/20_recipes/004_cli
 /usr/local/lib/python3.7/site-packages/optuna/multi_objective/visualization/_pareto_front.py:docstring of optuna.multi_objective.visualization.plot_pareto_front:7: WARNING: Unknown directive type "plotly". 

```

Reply:
Let us work on this in a follow-up PR.